.gitignore

```diff
@@ -39,6 +39,6 @@ tests/testing-app
 # drone CI is in .drone.star, do not let someone accidentally commit a local .drone.yml
 .drone.yml
 
-# Themes
-/packages/web-runtime/themes/**
-!/packages/web-runtime/themes/owncloud/**
+# # Themes
```

Comment:
@elizavetaRa @diocas please note that neither you nor us want the CERN theme in the ownCloud web master, esp. regarding licensing and logo files etc. Could you drop the respective commits from your fork in this PR? :)

Reply:
This was not supposed to have happened. Those commits are only for our fork and are temporary.

Reply:
@elizavetaRa please keep only your commits and rebase on the upstream, not on our fork.

Reply:
No bad intent assumed ;) ping me/us if you need any suppor there

Reply:
@pascalwengerter in future we will do the pull requests from cernbox/web, I closed this and opened the corresponding one with newest changes: https://github.com/owncloud/web/pull/5177